jhipster/src/main/webapp/content/css/patient.css

```diff
@@ -1,37 +1,90 @@
-.container {
-    margin-top: 5%;
-    overflow: hidden;
+.container_info {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
 }
 
-.logo{
-    width: 200px;
-    height: 200px;
+.flex_div {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
 }
 
-.fixed-flex-container {
+.photo_patient{
+    width: 80px;
+    height: 80px;
+}
+
+.info_patient_administratives {
     display: flex;
     flex-wrap: wrap;
     justify-content: space-between;
+    flex-direction: column;
+    margin-left: 5px;
+    height: 150px;
 }
 
-.jh-entity-details {
+.info-card-red {
```

Comment:
create a class info-card and then decline it in subclass red, orange and blue

Reply:
.info-card{    
    color: white;
    display: flex;
    border: 2px solid #000;
    margin: 0px;
}

.info-card.red {
    background-color: rgb(238, 116, 116);
}

.info-card.orange { ...